scipy/signal/windows/windows.py

```diff
@@ -1640,6 +1641,60 @@ def exponential(M, center=None, tau=1., sym=True):
     return _truncate(w, needs_trunc)
 
 
+def taylor(N, nbar=4, sll=-30):
+    """
+    Return the Taylor window.
+
+    The Taylor window allows for a selectable sidelobe suppression with a 
+    minimum broadening. This window is commonly used in synthetic aperture radar
+    processing.
+
+    Parameters
+    ----------
+    M : int
+        Number of points in the output window. If zero or less, an
+        empty array is returned.
+    nbar : int
+        Number of nearly constant level sidelobes adjacent to the mainlobe
+    sll : float
+        Desired peak sidelobe level in decibels (db) relative to the mainlobe
+
+    Returns
+    -------
+    out : array
+        The window, with the maximum value normalized to one (the value
+        one appears only if the number of samples is odd).
+
+    See Also
+    --------
+    kaiser, bartlett, blackman, hamming, hanning
+
+    Notes
+    -----
+    The Kaiser window is described by Carrara, Goodman, and Majewski 
```

Comment:
Why is this about `Kaiser`?

Can you add a reference for the Taylor window, and put it in a `References` section, and refer to it with a `[1]_` above (see how we do it elsewhere)?